contracts/test-utils/src/vote_factory.ts

```diff
@@ -0,0 +1,79 @@
+import { assert } from '@0x/assert';
+import { schemas } from '@0x/json-schemas';
+import { eip712Utils } from '@0x/order-utils';
+import { ECSignature, EIP712DomainWithDefaultSchema, EIP712TypedData, SignatureType } from '@0x/types';
+import { BigNumber, signTypedDataUtils } from '@0x/utils';
+import * as ethUtil from 'ethereumjs-util';
+import * as _ from 'lodash';
+
+import { constants } from './constants';
+import { signingUtils } from './signing_utils';
+import { SignedZeroExVote, ZeroExVote } from './types';
+
+export class VoteFactory {
+    private readonly _delegatorPrivateKey: Buffer;
+    private readonly _domain: EIP712DomainWithDefaultSchema;
+
+    public static newZeroExVote(
+        proposalId: BigNumber,
+        support: boolean,
+        operatedPoolIds: number[],
+    ): ZeroExVote {
+        return {
+            proposalId,
+            support,
+            operatedPoolIds,
+        };
+    }
+
+    constructor(
+        delegatorPrivateKey: Buffer,
+        contractName: string,
+        chainId: number,
+        verifyingContract: string,
+    ) {
+        this._delegatorPrivateKey = delegatorPrivateKey;
+        this._domain = {name: contractName, chainId, verifyingContract};
+    }
+
+    public async newSignedZeroExVoteAsync(
+        vote: ZeroExVote,
+        signatureType: SignatureType = SignatureType.EIP712,
+    ): Promise<SignedZeroExVote> {
+        const voteHashBuffer = this._getVoteHashBuffer(vote);
+        const signature = signingUtils.signMessage(voteHashBuffer, this._delegatorPrivateKey, signatureType);
```

Comment:
You can use [this function](https://github.com/0xProject/protocol/blob/development/packages/protocol-utils/src/signature_utils.ts#L112) to sign, that way you won't need to export `parseSignatureHexAsRSV`